README.md

```diff
@@ -43,11 +43,26 @@ For more information on [how to get involved on the CHAOSS website](https://chao
 
 ## Collecting Data
 
-Augur supports ```Python3.7``` through ```Python3.11``` on all platforms. ```Python3.12``` and above do not yet work because of machine learning worker dependencies. On OSX, you can create a ```Python3.11``` environment, by running:
+Augur aims to support the current officially supported Python versions (currently centered around **Python 3.11**). We recommend using a Python installation that matches your platform's supported releases and creating an isolated virtual environment for development.
+
+Quick virtual environment examples:
+
+Windows (PowerShell):
+```
+python -m venv .venv
+.\.venv\Scripts\Activate.ps1
 ```
-$ python3.11 -m venv path/to/venv
+
+macOS / Linux:
+```
+python3 -m venv .venv
+source .venv/bin/activate
 ```
 
+After activating the virtual environment, install the project's development dependencies as described in the developer docs.
+
+For full, up-to-date setup instructions (including Docker-based development), see the developer documentation: https://oss-augur.readthedocs.io/en/main/development-guide/toc.html
+
```

Comment:
I like the way this starts out in terms of talking about the python support, but I also liked @cnaples79's version (#3443) which says explicitly that augur uses UV for managing python dependencies.

Uv as a dependency manager doesnt really have a specific "activation" process, you just run commands through `uv run` and it handles it all for you, including downloading and setting up the virtual env behind the scenes.